scripts/DDL.sql

```diff
@@ -1,48 +1,75 @@
+CREATE SCHEMA jobs;
 
-CREATE TABLE users(
+CREATE TABLE jobs.users(
 	upn text PRIMARY KEY,
 	display_name text
 );
 
-CREATE TABLE tags(
+CREATE TABLE jobs.tags(
 	id serial PRIMARY KEY,
 	name text
 );
 
-CREATE TABLE roles(
+CREATE TABLE jobs.roles(
 	id serial PRIMARY KEY,
 	name text
 );
 
-CREATE TABLE standards(
+CREATE TABLE jobs.standards(
 	id serial PRIMARY KEY,
 	name text
 );
 
-CREATE TABLE advertisements(
+CREATE TABLE jobs.units(
 	id serial PRIMARY KEY,
-	role_id int REFERENCES roles(id) ON DELETE CASCADE,
-	unit text,
-	branch text,
-	department text,
+	name text UNIQUE
+);
+
+CREATE TABLE jobs.branches(
+	id serial PRIMARY KEY,
+	unit_id int REFERENCES jobs.units(id) ON DELETE CASCADE,
+	name text UNIQUE,
```

Comment:
`name` should be defined **before** `foreign key`... Same in line: 38

Reply:
ok will be done